domains/Contact/ManageTasks/Web/ViewHelpers/ModuleContactTasksViewHelper.php

```diff
@@ -57,6 +57,8 @@ public static function dtoTask(Contact $contact, ContactTask $task, User $user):
             'description' => $task->description,
             'completed' => $task->completed,
             'completed_at' => $task->completed_at ? DateHelper::format($task->completed_at, $user) : null,
+            'due_at' => $task->due_at ? DateHelper::format($task->due_at, $user) : null,
+            'due_at_late' => $task->due_at ? $task->due_at->isPast() : false,
```

Comment:
You can use the `optional` helper to write:
```suggestion
            'due_at_late' => optional($task->due_at)->isPast() ?? false,
```